compile.sh

```diff
@@ -6,6 +6,7 @@
 # Update jq repo and its submodules
 cd jq
 git submodule update --init
+git submodule update --init
```

Comment:
This looks like a duplicate line :)

Reply:
Actually that's very intentional - it didn't work when I only had one. There seem to be two updates, and if you only run it once, you only get the first (and the build fails).

Reply:
Huh interesting

Reply:
It could be that we could try `git submodule update --init --recursive` but I figure the current works, so it's maybe not worth the extra testing.